pkg/clients/eks/eks.go

```diff
@@ -105,9 +106,9 @@ type AMIClient interface {
 }
 
 type eksClient struct {
-	eks            eksiface.EKSAPI
+	eks            eksiface.ClientAPI
 	amiClient      AMIClient
-	sts            *sts.STS
+	sts            *sts.Client
```

Comment:
Ditto here - what's the context for this change?

Reply:
Same as above